### PR TITLE
WebAPI: Update Method pages to modern structure (part 35)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
### Summary
Updates `### Parameters` sections.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/createimagebitmap/index.md [hunk 1]
```diff
@@ -52,19 +52,19 @@ createImageBitmap(image, sx, sy, sw, sh, options)
 
   - : An object that sets options for the image's extraction. The available options are:
 
-    - `imageOrientation`: Specifies whether the image should be presented
+    - `imageOrientation`:  Specifies whether the image should be presented
```
Comment (teoli2003) [inline files/en-us/web/api/createimagebitmap/index.md:55]: This one is incorrect (should be a definition list but if it is outside the scope of this PR, it should at least not add an extra space)

File: files/en-us/web/api/cssnumericvalue/add/index.md [hunk 2]
```diff
@@ -21,13 +21,12 @@ The **`add()`** method of the
...
-add(double)
-add(CSSNumericValue)
+add(number)
 ```
 
 ### Parameters
 
-- number
+- `number`
   - : either a {{jsxref('Number')}} or a {{domxref('CSSNumericValue')}}.
```
Comment (teoli2003) [inline files/en-us/web/api/cssnumericvalue/add/index.md:30]:
```suggestion
  - : Either a {{jsxref('Number')}} or a {{domxref('CSSNumericValue')}}.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I'm really excited about these changes. They are the final piece to get most API reference pages up to the current structure.


Two small nits to update before we can land this.

**Review by teoli2003 — APPROVED** [review]
\o/

File: files/en-us/web/api/createimagebitmap/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/createimagebitmap/index.md]: > it should at least not add an extra space
It is a tagging mechanism I use in certain cases where manually fixing stuff is quicker than writing/testing the code for it. 🤓

**Comment by OnkarRuikar** [comment]
> I'm really excited about these changes. They are the final piece to get most API reference pages up to the current structure.
This would've been complete in the last week. But DOMString PRs were creating a lot of conflicts, so I completed them first.

**Comment by teoli2003** [comment]
The project is 17 years old, we have time :-)

**Pull request merged**